indico/modules/events/papers/client/js/components/GroupReviewForm.jsx

```diff
@@ -0,0 +1,209 @@
+          value="yes"
+        />
+        <FinalRadio
+          name={fieldName}
+          required={isRequired}
+          label={Translate.string('No')}
+          value="no"
+        />
+      </Form.Group>
+    );
+  } else if (fieldType === 'rating') {
+    return (
+      <FinalRating name={fieldName} required={isRequired} max={maxScore} min={minScore} allowNull />
+    );
+  }
+}
+
+ReviewingQuestion.propTypes = {
+  question: PropTypes.object.isRequired,
+};
+
+/**
+ *
+ * @param {Object} group - object with the info about the reviewing group
+ * @param {Object} review - object with the data used to prefill the form's initial values
+ * @param {Function} onSubmit - function invoked on the form submission
+ * @param {Function} onCancel - function invoked once the user decides to stop filling in the form
+ */
+export default function GroupReviewForm({group, review, onSubmit, onCancel}) {
+  const questions = useSelector(getReviewingQuestions)[group.name];
+  const renderQuestions = () => {
+    return questions.map((question, index) => {
+      return (
+        <Popup
+          key={question.id}
+          position="left center"
+          content={question.description}
+          disabled={!question.description}
+          trigger={
+            <div className="flexrow question-row">
+              <div>
+                <span className="question-index">{index + 1}</span>
+              </div>
+              <div className="question-text f-self-stretch">
+                {question.title}
+                {question.isRequired && (
+                  <Popup
+                    trigger={<i className="question-required"> *</i>}
+                    content={Translate.string('Rating this question is mandatory')}
+                    position="bottom center"
+                  />
+                )}
+              </div>
+              <div styleName="question-field">
+                <ReviewingQuestion question={question} />
+              </div>
+            </div>
+          }
+        />
+      );
+    });
+  };
+
+  const initialValues = {};
+  if (review) {
+    initialValues.proposed_action = review.proposedAction.name;
+    initialValues.comment = review.comment;
+
+    for (const {value, question} of review.ratings) {
+      if (question.fieldType === 'bool') {
+        initialValues[`question_${question.id}`] = value ? 'yes' : 'no';
+      } else {
+        initialValues[`question_${question.id}`] = value;
+      }
+    }
+  }
+
+  return (
+    <FinalForm onSubmit={onSubmit} subscription={{submitting: true}} initialValues={initialValues}>
+      {fprops => (
+        <Form onSubmit={fprops.handleSubmit} styleName="group-review-form">
+          {!review && (
+            <>
+              <div className="form-preface">
+                <Translate>
+                  Reviewing in{' '}
+                  <Param
+                    name="group"
+                    value={<span>{group.title}</span>}
```

Comment:
is that span needed?

Reply:
Hm, probably not. Before it used to have a tooltip attached but I removed it and probably forgot to remove the `span` altogether.